test/fast-list-test.js

```diff
@@ -149,11 +92,11 @@ suite('FastList >', function() {
 
       test('it renders a whole prerendered viewport of list items', function() {
         // 7.5 per screen * 2.8 (will-change budget) -> 21
-        assert.equal(container.querySelectorAll('ul li').length, 21);
+        assert.equal(container.querySelectorAll('ul li').length, 22);
       });
 
       test('it renders the correct content', function() {
-        assertCurrentlyRenderedWindow(0, 20);
+        assertCurrentlyRenderedWindow(container, source, 0, 21);
```

Comment:
NIT: When the number of arguments > 2, I'd prefer to see a single object passed to help with readability. Otherwise I have to continually refer to the function definition to work out what each argument is.